python/uagents-adapter/pyproject.toml

```diff
@@ -0,0 +1,66 @@
+[tool.poetry]
+name = "uagents-adapter"
+version = "0.1.0"
+description = "Adapters for uAgents to integrate with LangChain and CrewAI"
+authors = ["Fetch.AI Limited"]
+license = "Apache 2.0"
+readme = "README.md"
+keywords = ["uagents", "agents", "langchain", "crewai", "fetch.ai"]
+packages = [{include = "uagents_adapter", from = "src" }]
+
+[tool.poetry.dependencies]
+python = "^3.10"
+uagents = "<=0.22.3"
+pydantic = ">=2.8,<3.0"
+langchain-core = ">=0.3.52,<0.4.0"
+requests = "^2.32.0"
+
+[tool.poetry.group.dev.dependencies]
+pytest = "^8.3.5"
+pytest-asyncio = "^0.25.3"
+ruff = "^0.11.0"
+pre-commit = "^4.1.0"
+
+[tool.poetry.group.langchain]
+optional = true
+
+[tool.poetry.group.langchain.dependencies]
+langchain-core = ">=0.3.52,<0.4.0"
+langchain-openai = ">=0.3.13,<0.4.0"
+
+[tool.poetry.group.crewai]
+optional = true
+
+[tool.poetry.group.crewai.dependencies]
```

Comment:
given if its installed as a package both can be imported in, I would just go with simple flat dependencies for easiness, and then ci doesn't need much update to do auto pypi release. what you think @jrriehl ?